src/web/components/SlotBodyEnd.tsx

```diff
@@ -67,51 +61,74 @@ export const SlotBodyEnd = ({
             referrerUrl: window.location.origin + window.location.pathname,
         },
         localisation: {
-            countryCode,
+            countryCode: props.countryCode,
         },
         targeting: {
-            contentType,
-            sectionName: sectionName || '', // TODO update client to reflect that this is optional
-            shouldHideReaderRevenue,
-            isMinuteArticle,
-            isPaidContent,
-            tags,
+            contentType: props.contentType,
+            sectionName: props.sectionName || '', // TODO update client to reflect that this is optional
+            shouldHideReaderRevenue: props.shouldHideReaderRevenue,
+            isMinuteArticle: props.isMinuteArticle,
+            isPaidContent: props.isPaidContent,
+            tags: props.tags,
             showSupportMessaging: shouldShowSupportMessaging(),
-            isRecurringContributor: isRecurringContributor(isSignedIn),
+            isRecurringContributor: isRecurringContributor(
+                props.isSignedIn || false,
+            ),
             lastOneOffContributionDate: getLastOneOffContributionDate(),
-            epicViewLog,
+            epicViewLog: getViewLog(),
+            mvtId: Number(getCookie('GU_mvt_id')),
         },
     };
+};
 
-    const getSlot: () => Promise<Response> = useCallback(
-        () => getBodyEnd(contributionsPayload),
-        [], // empty as we only want to call the API once and payload will always fail a reference equality check anyway
-    );
-
-    const { data: bodyResponse, error } = useApiFn<{
-        data: { html: string; css: string };
-    }>(getSlot);
-
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'slot-body-end');
-    }
+const MemoisedInner = (props: Props) => {
+    const [data, setData] = useState<{
+        slot?: {
+            html: string;
+            css: string;
+        };
+    }>();
 
-    if (bodyResponse && bodyResponse.data) {
-        // Add a new entry to the view log when we know an Epic is being rendered
-        logView(epicTestName);
-        sendOphanInsertEvent();
+    useEffect(() => {
+        const contributionsPayload = buildPayload(props);
+        getBodyEnd(contributionsPayload)
+            .then(response => response.json())
+            .then(checkForErrors)
+            .then(json =>
+                setData({
+                    slot: { html: json.data.html, css: json.data.css },
+                }),
+            )
+            .then(_ => sendOphanInsertEvent())
```

Comment:
Since I don't think we have to worry about the binding here, what about:

```suggestion
            .then(sendOphanInsertEvent)
```